doc/mgr/rgw.rst

```diff
@@ -75,3 +75,32 @@ Update the realm configuration to match the orchestrator deployment
   ceph rgw admin [*]
 
 RGW admin command
+
+Automatic usage log trimming
+----------------------------
+
+Enable automatic usage log trimming for one or more RGW realms by
+setting the config option ``usage_trim_realms`` to a comma separated
+list of realms.
+
+Set the ``usage_trim_older_than_days`` config option to the amount of
+days usage logs older than this should be trimmed. Optionally you can
+set the interval that trim will be performed with ``usage_trim_interval``,
+it defaults to every twelve hours.
```

Comment:
I think we should specify the units of this value (seems to be seconds! do we really need that fine granularity?)

Reply:
I will fix this! I think it's kind of hard to anticipate what people want to use it for, could potentially change to minutes but not hours, what do you think?

Reply:
The option `usage_trim_older_than_days` uses days as units so I don't see what the benefit of proving more granularity for the option `usage_trim_interval`. I think hours should be OK unless you have some use cases in mind when this wouldn't be enough.

Reply:
I've made it more clear that it's seconds for the interval and I think the default (12 hours in seconds) is a sensible default that nobody probably wants (needs?) to change but we provide it for convenience either way.